command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 04:32:58PM -0700, Mike Brown wrote:
> Ask apple it seems to work for their app certification process.

I skimmed through [1,2] and don't see anything about source
inspection.  It looks like Apple does reserve the right to reject
submissions for a number of subjective criteria (e.g. “Not enough
lasting value” [3]), though and that's the same class of issues.  They
provide a resolution center and appeal board [1] to resolve disputes
(with the latter presumably focused mostly on the subjective
decisions).

My impression of the process is that it is going to be something that
makes more sense for a monolithic company like Apple than it does for
a multi-member organization like the OCI, and that it would require a
lot of compliance-tester time that has not currently materialized in
OCI-land.  I don't get the impression that we're headed towards
subjective certification now, but that doesn't mean that there
wouldn't be significant OCI-compliance funding to move in that
direction in the future.  My preference would be to not move in that
direction ;).

[1]: https://developer.apple.com/support/app-review/
[2]: https://developer.apple.com/app-store/review/guidelines/
[3]: https://developer.apple.com/app-store/review/rejections/